cmd_states/run_test.py

```diff
@@ -0,0 +1,2 @@
+import Chandra.cmd_states
+Chandra.cmd_states.test()
```

Comment:
@jzuhone  Regarding cmd_states, it looks like the last two releases didn't get git tags, so I've just pushed the last one.   Also when building for conda I get test errors on  looking for "SKA" (as not set).  Are those test failures unique to my setup?  I don't see anything in your build/test or scripts setting SKA.  It looks like 'conda-build' does seem to ignore the test failures and continue though.  


```
>           raise KeyError(key) from None
E           KeyError: 'SKA'

../../os.py:669: KeyError
================================================================================== 3 failed in 7.33 seconds ==================================================================================
===== cmd_states-3.11-py36_0 OK =====
import: 'Chandra.cmd_states'
TEST END: /data/fido/c3/conda-bld/linux-64/cmd_states-3.11-py36_0.tar.bz2
INFO:conda_build.config:--dirty flag not specified.  Removing build folder after successful build/test.
```